geocity/apps/submissions/models.py

```diff
@@ -1517,6 +1517,12 @@ class SubmissionGeoTime(models.Model):
     comes_from_automatic_geocoding = models.BooleanField(
         _("Géométrie obtenue par géocodage d'adresse"), default=False
     )
+    form = models.ForeignKey(
+        Form, on_delete=models.CASCADE, related_name="geo_time", null=True
+    )
+    field = models.ForeignKey(
+        Field, on_delete=models.CASCADE, related_name="geo_time", null=True
+    )
```

Comment:
I have a doubt about this one. Should be discussed

Reply:
This means an instance of the `SubmissionGeoTime `model can be - but does not have to - be associated with a `Field `and a `Form `model instance. On detail step submission, if geometry fields are used, form and field reference are saved: https://github.com/yverdon/geocity/pull/870/files/0fd917fc03febbed9d421fc8c7c37aa7217bd472#diff-1d617db03e3ebd04cf98c37e14ede1e17b638c6f29475e0abbab6bd31535eb91R754-R760